pkg/scheduler/scheduler.go

```diff
@@ -74,6 +75,7 @@ type Scheduler struct {
 	clusterLister        clusterlister.ClusterLister
 	informerFactory      informerfactory.SharedInformerFactory
 
+	clusterWorker util.AsyncWorker
```

Comment:
It'd be better to have a clear comment.
```suggestion
	// ClusterReconcileWorker reconciles cluster changes to trigger corresponding
	// ResourceBinding/ClusterResourceBinding rescheduling.
	ClusterReconcileWorker util.AsyncWorker
```